index.bs

```diff
@@ -136,6 +136,8 @@ In addition to the Image Properties defined in [[!HEIF]], such as 'colr', 'pixi'
 
 In general, it is recommended to use properties instead of [=Metadata OBUs=] in the [=AV1 Item Configuration Property=].
 
+For multi-layer images, the LayerSelectorProperty may be used and if used, it shall be as defined in [[!HEIF]].
```

Comment:
Should we say "may be _present_" instead of "may be _used_"? I think the term "used" is a bit vague -- it is not clear if it refers to the encoder or the parser/reader. By changing the word to "present" I think it becomes clear that the encoder may add this box. We don't say anything about whether a parser/reader must support the box or not, which I think is OK.

Reply:
I agree my proposal is a bit vague. Actually, I wasn't clear about the use of the LayerSelectionProperty, that's probably why. Reading the HEIF spec again, I see:

> The decoding of a multi-layer image item may result into one or more reconstructed images.

> NOTE 1: For example, when a multi-layer image contains two views, a decoder is typically specified to decode and return the decoded images of both the views.

> When the decoding of a multi-layer image item results into more than one reconstructed images, a LayerSelectorProperty item property shall be present for the image item.

What I understand is:
- if you have a multi-layered image but in the traditional sense (i.e. the next layer improves the quality/resolution of the previous image), then you don't have to put the LayerSelectorProperty.
- if you have a multi-layered image for the multi-view case, you have to put the LayerSelectorProperty.

Is that correct @dwsinger?

The question for AV1 is for AV1: Is the notion of producing multiple reconstructed images applicable? 
I think it is, but I'm not sure. @agrange ? Can we have a base layer used by 2 enhancement layers that are not related to each other, but only predicting from the base?

Reply:
Yes, there can be multiple unrelated enhancement layers that refer to the same base layer.